ci/build_container/build_container.sh

```diff
@@ -32,10 +26,6 @@ pip install virtualenv
 export GOPATH=/usr/lib/go
 go get github.com/bazelbuild/buildifier/buildifier
 
-# GCC 4.9 for everything.
-export CC=gcc-4.9
-export CXX=g++-4.9
-
```

Comment:
Is it worth invoking gcc here for a version check to ensure we have the version we think we do?